### DISCUSSION TITLE
Changing background color of an MDX story in Storybook

### DISCUSSION THREAD

**Discussion by nareshbhatia** [storybookjs/storybook#14661] [Help]
I would like to change the background color of a story in Storybook. The story is written in MDX. Here's what I've tried but it does not work - the background remains white.

```
<Meta
  title="Style Guide/Colors"
  parameters={{
    backgrounds: {
      default: 'palette',
      values: [
        {
          name: 'palette',
          value: '#e5e5e5',
        },
      ],
    },
  }}
/>
```

![image](https://user-images.githubusercontent.com/1220198/115342466-550d1400-a178-11eb-9479-1a72f582294e.png)

What am I missing?

Please note that I don't want the user to be able to change the background color on this story. Hence I have specified only one option.

Full code for the story can be found in my project [here](https://github.com/PublicisSapient/accelerated-news/blob/main/src/stories/style-guide/Colors.stories.mdx).

**Comment by shilman** [comment]
Maybe dupe to this https://github.com/storybookjs/storybook/issues/13323

**Comment by rahuldkjain** [comment]
@nareshbhatia You can put inline styles in Canvas. You can achieve the desired background like this.
```jsx
<Canvas style={{ background: '#000000' }}>
    <Story
        name="Button"
        args={{
            text: "Click me"
        }}
    >
        {Button.bind({})}
    </Story>
</Canvas>

**Reply by joeyfigaro** [reply]
For anyone else landing here, this does not currently work–also, Canvas wraps individual stories so you'd only get colored backgrounds for specific stories instead of the entire page.
